### DISCUSSION TITLE
为什么小程序vue模式下原生组件editor无法显示

### DISCUSSION THREAD

**Discussion by pnchen** [NervJS/taro#10180] [Q & A]
例如：
```
<view class="container">
	<editor id="editor" class="editor" :placeholder="placeholder" @ready="editorReady"></editor>
	<button type="warn" @tap="undo">撤销</button>
</view>
```
页面提示
![image](https://user-images.githubusercontent.com/2562679/131810883-f5daf8f9-0490-4f8f-9788-3fb8c8486d0e.png)

Taro版本 3.3.5

**Comment by Chen-jj** [comment]
@pnchen Bug Report 提 issue 咯，需要 issue 模板提供的详细信息才能定位。
